.mailmap

```diff
@@ -413,6 +456,7 @@ Varun Nayyar <nayyarv@gmail.com> <nayyarv@users.noreply.github.com>
 Vrinda Narayan <talk2vrinda@gmail.com> <vrinda18120@iiitd.ac.in>
 Vrinda Narayan <talk2vrinda@gmail.com> <48102157+vrindaaa@users.noreply.github.com>
 Wansoo Kim <rladhkstn8@gmail.com>
+Warren Weckesser <warren.weckesser@gmail.com>
```

Comment:
Warren is in twice, if that matters.

Reply:
So are others. The first maps the email address to a name, regardless of the commit name. The second maps a different email address to the name and address. It would be nice if only one entry were needed, but it doesn't work that way, I tried it.